docs/README.md

```diff
@@ -2,8 +2,13 @@
 Triggers enables users to map fields from an event payload into resource templates. Put another way, this allows events to both model and instantiate themselves as Kubernetes resources. In the case of `tektoncd/pipeline`, this makes it easy to encapsulate configuration into `PipelineRun`s and `PipelineResource`s. 
 
 ![TriggerFlow](../images/TriggerFlow.png)
-# Learn more
-See the following topics for more on each of the resources involved:
+
+## Learn More
+See the following links for more on each of the resources involved:
 - [`TriggerTemplate`](triggertemplates.md)
 - [`TriggerBinding`](triggerbindings.md)
 - [`EventListener`](eventlisteners.md)
+
+## Getting Started Tasks
```

Comment:
Seems like we can remove this list, and just link to the getting started doc here instead here in that PR. What do you think?

Reply:
I agree!